src/commands/test/lib/xml-util.ts

```diff
@@ -5,49 +5,56 @@ export abstract class XmlUtil {
   public abstract mergeXmlResults(pathToArchive: string): Promise<Document>;
   public abstract getArchiveName(): string;
 
-  public collectAllElements(node: Node, name: string): Node[] {
-    let result: Node[] = [];
-    if (!node || !name) {
+  public collectAllElements(element: Element, name: string): Element[] {
+    let result: Element[] = [];
+    if (!element || !name) {
       return result;
     }
-    if (node.nodeName === name) {
-      result.push(node);
+    if (element.nodeName === name) {
+      result.push(element);
     }
-    for (let i: number = 0; node.childNodes && i < node.childNodes.length; ++i) {
-      result = result.concat(this.collectAllElements(node.childNodes[i], name));
+    for (let i: number = 0; element.childNodes && i < element.childNodes.length; ++i) {
+      result = result.concat(this.collectAllElements(element.childNodes[i] as Element, name));
     }
     return result;
   }
 
-  public collectChildren(node: Node, name: string): Node[] {
-    if (!node || !name) {
+  public collectChildren(element: Element, name: string): Element[] {
+    if (!element || !name) {
       return [];
     }
-    if (node.nodeName === name) {
-      return [node];
+    if (element.nodeName === name) {
+      return [element];
     }
-    let result: Node[] = [];
-    for (let i: number = 0; node.childNodes && i < node.childNodes.length; ++i) {
-      result = result.concat(this.collectChildren(node.childNodes[i], name));
+    let result: Element[] = [];
+    for (let i: number = 0; element.childNodes && i < element.childNodes.length; ++i) {
+      result = result.concat(this.collectChildren(element.childNodes[i] as Element, name));
     }
     return result;
   }
 
-  public countChildren(node: Node): number {
-    if (!node || !node.childNodes) {
+  public countChildren(element: Element): number {
+    if (!element || !element.childNodes) {
       return 0;
     }
-    let result = node.childNodes.length;
-    for (let i: number = 0; i < node.childNodes.length; ++i) {
-      result += this.countChildren(node.childNodes[i]);
+    let result = element.childNodes.length;
+    for (let i: number = 0; i < element.childNodes.length; ++i) {
+      result += this.countChildren(element.childNodes[i] as Element);
     }
     return result;
   }
 }
 
 export function validXmlFile(file: string): boolean {
   try {
-    const xml = new DOMParser().parseFromString(fs.readFileSync(file, "utf-8"));
+    const configuration: object = {
```

Comment:
Can we share this configuration somewhere so it's defined in one place?

Reply:
I will refactor this as well